### PR TITLE
translate 2020-10-01

### PR DESCRIPTION
**Pull request by ldsdsy**
I noticed that there is no Chinese equivalent for this section, so I have translated it.

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
@ldsdsy
Pls sign the CLA in order for the PR to be reviewed.
more info
https://github.com/kubernetes/community/blob/master/CLA.md

**Comment by ldsdsy** [comment]
@Arhell 

image

I received the email that the signing was successful, but the status of the robot did not change. Why is this?

File: content/zh/blog/_posts/2020-10-01-contributing-to-the-development-guide/index.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+linkTitle: "为发展指南做贡献"
+Author: Erik L. Arneson
+Description: "一位新的贡献者描述了对 Kubernetes 开发指南进行修改的编写和提交经验。"
+date: 2020-10-01
+canonicalUrl: https://www.kubernetes.dev/blog/2020/09/28/contributing-to-the-development-guide/
+resources:
+- src: "jorge-castro-code-of-conduct.jpg"
+  title: "Jorge Castro 正在 SIG ContribEx 的周例会上宣布 Kubernetes 的行为准则。"
+---
+
```
Comment (tengqm) [inline content/zh/blog/_posts/2020-10-01-contributing-to-the-development-guide/index.md:23]: 第12-22行必须放到文件头。

File: content/zh/blog/_posts/2020-10-01-contributing-to-the-development-guide/index.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+canonicalUrl: https://www.kubernetes.dev/blog/2020/09/28/contributing-to-the-development-guide/
+resources:
+- src: "jorge-castro-code-of-conduct.jpg"
+  title: "Jorge Castro announcing the Kubernetes Code of Conduct during a weekly SIG ContribEx meeting."
+--- -->
+---
+title: "为发展指南做贡献"
+linkTitle: "为发展指南做贡献"
+Author: Erik L. Arneson
+Description: "一位新的贡献者描述了对 Kubernetes 开发指南进行修改的编写和提交经验。"
```
Comment (tengqm) [inline content/zh/blog/_posts/2020-10-01-contributing-to-the-development-guide/index.md:16]:
```suggestion
Description: "一位新的贡献者描述了编写和提交对 Kubernetes 开发指南的修改的经验。"
```

File: content/zh/blog/_posts/2020-10-01-contributing-to-the-development-guide/index.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+date: 2020-10-01
+canonicalUrl: https://www.kubernetes.dev/blog/2020/09/28/contributing-to-the-development-guide/
+resources:
+- src: "jorge-castro-code-of-conduct.jpg"
+  title: "Jorge Castro 正在 SIG ContribEx 的周例会上宣布 Kubernetes 的行为准则。"
+---
+
+<!-- When most people think of contributing to an open source project, I suspect they probably think of
```
Comment (tengqm) [inline content/zh/blog/_posts/2020-10-01-contributing-to-the-development-guide/index.md:24]: 所有 `<!--` 和 `-->` 都应放在独立的一行。

File: content/zh/blog/_posts/2020-10-01-contributing-to-the-development-guide/index.md [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+contributing code changes, new features, and bug fixes. As a software engineer and a long-time open
+source user and contributor, that's certainly what I thought. Although I have written a good quantity
+of documentation in different workflows, the massive size of the Kubernetes community was a new kind 
+of "client." I just didn't know what to expect when Google asked my compatriots and me at
+[Lion's Way](https://lionswaycontent.com/) to make much-needed updates to the Kubernetes Development Guide.
+
+*This article originally appeared on the [Kubernetes Contributor Community blog](https://www.kubernetes.dev/blog/2020/09/28/contributing-to-the-development-guide/).* -->
+
+当大多数人想到为一个开源项目做贡献时，我猜想他们可能想到的是贡献代码修改、新功能和错误修复。作为一个软件工程师和一个长期的开源用户和贡献者，这也正是我的想法。虽然我已经在不同的工作流中写了不少文档，但规模庞大的 Kubernetes 社区是一种新型 "客户端"。我只是不知道当 Google 要求我和 [Lion's Way](https://lionswaycontent.com/) 的同胞们对 Kubernetes 开发指南进行急需更新时会发生什么。
```
Comment (tengqm) [inline content/zh/blog/_posts/2020-10-01-contributing-to-the-development-guide/index.md:33]: 对过长的行，要求适当折行。例如：

```
当大多数人想到为一个开源项目做贡献时，我猜想他们可能想到的是贡献代码修改、新功能和错误修复。
作为一个软件工程师和一个长期的开源用户和贡献者，这也正是我的想法。
虽然我已经在不同的工作流中写了不少文档，但规模庞大的 Kubernetes 社区是一种新型 "客户"。
我只是不知道当 Google 要求我和 [Lion's Way](https://lionswaycontent.com/) 的同胞们
对 Kubernetes 开发指南进行必要更新时会发生什么。
```

File: content/zh/blog/_posts/2020-10-01-contributing-to-the-development-guide/index.md [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+
+The Development Guide needed a pretty serious overhaul. When we got our hands on it, it was already
+packed with information and lots of steps for new developers to go through, but it was getting dusty
+with age and neglect. Documentation can really require a global look, not just point fixes.
+As a result, I ended up submitting a gargantuan pull request to the
+[Community repo](https://github.com/kubernetes/community): 267 additions and 88 deletions. -->
+
+## 这并不意味着这一切都很简单
+
+开发指南需要一次相当严重的整改。当我们拿到它的时候，它已经捆绑了大量的信息和很多新开发者需要经历的步骤，但随着时间的推移和被忽视，它变得相当陈旧。文档的确需要全局观，而不仅仅是点与点的修复。结果，最终我向这个项目提交了一个巨大的 pull 请求。[社区回帖](https://github.com/kubernetes/community)：新增 267 个，删除 88 个。
```
Comment (tengqm) [inline content/zh/blog/_posts/2020-10-01-contributing-to-the-development-guide/index.md:102]: ”相当严重的整改“，读起来不对

Comment (tengqm) [reply content/zh/blog/_posts/2020-10-01-contributing-to-the-development-guide/index.md:102]: community repo -> 社区仓库
新增 267 行
删除 88 行

Comment (ldsdsy) [reply content/zh/blog/_posts/2020-10-01-contributing-to-the-development-guide/index.md:102]: “一次大规模的整改”可以吗

**Comment by ldsdsy** [comment]
我已经修改完了，感谢您的建议，再次麻烦您审查一次。

File: content/zh/blog/_posts/2020-10-01-contributing-to-the-development-guide/index.md [hunk 6]
```diff
@@ -0,0 +1,196 @@
+---
+title: "为发展指南做贡献"
```
Comment (tengqm) [inline content/zh/blog/_posts/2020-10-01-contributing-to-the-development-guide/index.md:2]: development -> 开发

下同。

File: content/zh/blog/_posts/2020-10-01-contributing-to-the-development-guide/index.md [hunk 7]
```diff
@@ -0,0 +1,196 @@
...
+contributing code changes, new features, and bug fixes. As a software engineer and a long-time open
+source user and contributor, that's certainly what I thought. Although I have written a good quantity
+of documentation in different workflows, the massive size of the Kubernetes community was a new kind 
+of "client." I just didn't know what to expect when Google asked my compatriots and me at
+[Lion's Way](https://lionswaycontent.com/) to make much-needed updates to the Kubernetes Development Guide.
+
+*This article originally appeared on the [Kubernetes Contributor Community blog](https://www.kubernetes.dev/blog/2020/09/28/contributing-to-the-development-guide/).*
+-->
+
+当大多数人想到为一个开源项目做贡献时，我猜想他们可能想到的是贡献代码修改、新功能和错误修复。作为一个软件工程师和一个长期的开源用户和贡献者，这也正是我的想法。
```
Comment (tengqm) [inline content/zh/blog/_posts/2020-10-01-contributing-to-the-development-guide/index.md:39]: 请全文重新折行。

File: content/zh/blog/_posts/2020-10-01-contributing-to-the-development-guide/index.md [hunk 8]
```diff
@@ -0,0 +1,196 @@
...
+<!--
+ One of the primary traits of the relationship between our writing and our traditional clients is that we
+always have one or two primary points of contact inside a company. These contacts are responsible
+for reviewing our writing and making sure it matches the voice of the company and targets the
+audience they're looking for. It can be stressful -- which is why I'm so glad that my writing
+partner, eagle-eyed reviewer, and bloodthirsty editor [Joel](https://twitter.com/JoelByronBarker)
+handles most of the client contact. 
+-->
+
+我们的编写和传统客户之间的关系有一个主要的特点，就是我们在一个公司里面总是有一两个主要的对接人。他们负责审查我们的编写，并确保书写内容符合公司的声明且对标于他们正在寻找的受众。这随之而来的压力--正好解释了为什么我很高兴我的写作伙伴、鹰眼审稿人同时也是嗜血编辑的 [Joel](https://twitter.com/JoelByronBarker) 处理了大部分的客户联系。
```
Comment (tengqm) [inline content/zh/blog/_posts/2020-10-01-contributing-to-the-development-guide/index.md:67]: writing 可译为”写作“，或者根据上下文译为”作品“、”文稿“

Comment (ldsdsy) [reply content/zh/blog/_posts/2020-10-01-contributing-to-the-development-guide/index.md]: 已修改，辛苦您再审查一次

**Pull request merged**